### PR TITLE
Login & Registration Page #28

### LINKED ISSUE #28: Login & Registration Page
- [ ] Google Login with Firebase Auth

### PR DESCRIPTION
**Pull request by MadhushaPrasad**

### CODE REVIEW

**Comment by MadhushaPrasad** [comment]
Screenshot (158)
Screenshot (154)
Screenshot (159)
Screenshot (160)
Screenshot (161)

**Comment by Bawanthathilan** [comment]
@MadhushaPrasad  remove purple thunder logo and right side illustrations

**Comment by Bawanthathilan** [comment]
> @MadhushaPrasad remove purple thunder logo and right side illustrations

vector - https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.vecteezy.com%2Fvector-art%2F359502-vector-purple-landscape-illustration&psig=AOvVaw3L4Mt_t8A837zoBJoaXBkd&ust=1624115504816000&source=images&cd=vfe&ved=0CAoQjRxqFwoTCIjtpvC7ofECFQAAAAAdAAAAABAJ

**Comment by MadhushaPrasad** [comment]
Okay

File: src/pages/LoginPage/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          <p>Not Registered yet?</p><a href="/">Create An Account</a>
+        </div>
+      </form>
+    </div>
+  </div>
+</div>
+< div
+className = "col-sm-12 col-md-6 col-lg-6 bg-warning" >
+
+  < /div>*/
```
Comment (sithumonline) [inline src/pages/LoginPage/LoginPage.jsx]: Please remove unwanted code blocks

File: src/App.tsx [hunk 2]
```diff
@@ -15,7 +13,7 @@ function App() {
     <div>
       <QueryClientProvider client={queryClient}>
         <Router history={history}>
-          <Navbar />
+          {/*<Navbar />*/}
```
Comment (sithumonline) [inline src/App.tsx]: 🐭

File: src/App.tsx [hunk 3]
```diff
@@ -26,7 +24,7 @@ function App() {
               />
             ))}
           </Switch>
-          <Footer />
+          {/*<Footer />*/}
```
Comment (sithumonline) [inline src/App.tsx]: 🐭

File: src/modules/HomeModules/UpComingEvents/UpComing.scss [hunk 4]
```diff
@@ -76,16 +76,16 @@
...
+//.btn {
+//  background-color: #2a265f;
+//  border: 0;
+//  border-radius: 50px;
+//  box-shadow: 0 10px 10px rgba(0, 0, 0, 0.2);
+//  color: #fff;
+//  font-size: 16px;
+//  padding: 12px 25px;
+//  letter-spacing: 1px;
+//}
```
Comment (sithumonline) [inline src/modules/HomeModules/UpComingEvents/UpComing.scss]: 🐭

**Review by sithumonline — COMMENTED** [review]
Remove commented code blocks and some  scss blocks look like not properly formatted 🤔

File: src/pages/LoginPage/LoginPage.jsx (hunk 1 above)
Comment (MadhushaPrasad) [reply src/pages/LoginPage/LoginPage.jsx]: Okay

File: src/modules/HomeModules/UpComingEvents/UpComing.scss (hunk 4 above)
Comment (chamodshehanka) [reply src/modules/HomeModules/UpComingEvents/UpComing.scss]: @MadhushaPrasad remove the commented code

**Review by chamodshehanka — CHANGES_REQUESTED** [review]

**Comment by Safnaj** [comment]
No Need Text labels, placeholders is enough i guess for email & password.

And better to modify the vector by adding the project name as "SLIIT RSVP Portal"

@Shehanka @Bawanthathilan

**Pull request merged**